src/buildercore/config.py

```diff
@@ -81,22 +81,28 @@ class ConfigurationError(Exception):
 ROOTLOG.setLevel(logging.DEBUG) # *default* output level for all LOGs
 
 # StreamHandler sends to stderr by default
-H1 = logging.StreamHandler()
-H1.setLevel(logging.INFO) # output level for *this handler*
-H1.setFormatter(CONSOLE_FORMAT)
+CONSOLE_HANDLER = logging.StreamHandler()
```

Comment:
I know this has been like this before. But side effects like changing the logging configuration should ideally not happen by just importing a module.

Reply:
I agree because it's global state, but previously found some ramifications that make it difficult to avoid.

Reply:
don't get me started on logging ...